tests/cpp/multidevice_transformer.h

```diff
@@ -45,6 +45,10 @@ class DistributedTransformer {
         kDropoutProb(dropout_prob),
         kSdpaProb(sdpa_dropout_prob) {}
 
+  void setupForward(
+      Fusion *fusion,
+      DataType dtype,
+      bool sequence_parallel = false);
```

Comment:
Nit: consider an enum similar to https://github.com/NVIDIA/Fuser/blob/e057a9bac33023a2a7c4ed2cc87ebe6fe3944d8f/tests/python/test_transformer_engine.py#L25-L29. This makes the code easier to maintain and extend e.g. when we do context parallelism. See https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/